lib/app-router.js

```diff
@@ -84,6 +87,40 @@ module.exports = function (router) {
 
         break;
 
+      case 'apex':
+        let message = 'Apex Scripts run.';
+        async.eachSeries(param, function iteratee(apex, callback) {
+          console.log(`running ${apex}`);
+          script = `${startingDirectory}cd ${directory};export FORCE_SHOW_SPINNER=;sfdx force:apex:execute -f ${apex}`;
+
+          commands.run(command, script, (commandResult, commandErr) => {
+            if (commandErr) {
+              console.log(commandErr);
+              callback(commandErr);
+
+            } else {
+              console.log(commandResult);
+              // doesn't seem to be returning until after everything else runs.  fix UI output
+              message = `${message}
+              \t${apex}`;
+              callback();
+            }
+          });
+        }, function(err){
+          if (err){
```

Comment:
Is this a new pattern for handling errors? If so, let's create a function out of it and use it consistently.

Reply:
it's only necessary for the async loop--basically, if anything in the iteration throws an error, we want to catch and pass back.

So yes, when I add package(s) and data import(s) I'll use the same pattern.